hw/ip_templates/rstmgr/rstmgr.core.tpl

```diff
@@ -10,7 +10,7 @@ virtual:
 filesets:
   files_rtl:
     depend:
-      - lowrisc:ip:alert_handler_component
+      - lowrisc:ip_interfaces:alert_handler_pkg
```

Comment:
This is a .tpl file: could you use ${instance_vlnv...} for this instead?

Reply:
Haha, you've probably already found that happens on a later commit ;)

In order to keep each commit focused, there needed to be some carrying of virtual dependencies until the commit that changed that specific template.